From 155a9ceabdeb0c2e561a95a1a20c46e0655dbf0c Mon Sep 17 00:00:00 2001
From: Heinrich Schuchardt <xypron.glpk@gmx.de>
Date: Sun, 4 Apr 2021 13:30:13 +0000
Subject: [PATCH 1/1] cmd: CMD_USB depends on USB

The usb command cannot be compiled with CONFIG_USB=n:

ld.bfd: cmd/usb.c:660: undefined reference to `usb_stop'
ld.bfd: cmd/usb.c:663: undefined reference to `usb_started'

Signed-off-by: Heinrich Schuchardt <xypron.glpk@gmx.de>
---
 cmd/Kconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/cmd/Kconfig b/cmd/Kconfig
index eff238cb38..863b7f9fda 100644
--- a/cmd/Kconfig
+++ b/cmd/Kconfig
@@ -1309,6 +1309,7 @@ config CMD_UNIVERSE
 
 config CMD_USB
 	bool "usb"
+	depends on USB
 	select HAVE_BLOCK_DEVICE
 	help
 	  USB support.
-- 
2.31.0

